TX.h

```diff
@@ -564,6 +715,76 @@ static inline void processSBUS(uint8_t c)
   lastSerialPPM = millis();
 }
 
+static inline uint16_t multiToPpm(uint16_t input) {
+  uint32_t value = input;
+  // Taken from https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Multiprotocol/Multiprotocol.h#L478-L483
+  value = (((value - 204) * 1000) / 1639) + 1000;
```

Comment:
The likely culprit. I think this should be changed to:

value = ((value * 1400) / 2048) + 800;